wnfs-common/src/blockstore.rs

```diff
@@ -22,7 +22,7 @@ pub trait BlockStore {
     async fn put_block(&mut self, bytes: Vec<u8>, codec: IpldCodec) -> Result<Cid>;
 
     async fn put_serializable<V: Serialize>(&mut self, value: &V) -> Result<Cid> {
-        let bytes = dagcbor::encode(&ipld_serde::to_ipld(value)?)?;
+        let bytes = dagcbor::encode(&value)?;
```

Comment:
I didn't touch that because `wnfs_common` still uses libipld components. Changing from `dagcbor::{encode, decode}` probably makes more sense after BlockStore ipldcodec refactor, then we can move away from libipld stuff.